opendbc/car/hyundai/tests/test_hyundai.py

```diff
@@ -44,17 +45,20 @@
 class TestHyundaiFingerprint:
   def test_feature_detection(self):
     # HDA2
-    for has_adas in (True, False):
-      car_fw = [CarParams.CarFw(ecu=Ecu.adas if has_adas else Ecu.fwdCamera)]
-      CP = CarInterface.get_params(CAR.KIA_EV6, gen_empty_fingerprint(), car_fw, False, False)
-      assert bool(CP.flags & HyundaiFlags.CANFD_HDA2) == has_adas
+    for hda2 in (True, False):
+      fingerprint = gen_empty_fingerprint()
+      if hda2:
+        cam_can = CanBus(None, False, fingerprint).CAM
+        fingerprint[cam_can] = [0x50, 0x110]  # HDA2 steering messages
+      CP = CarInterface.get_params(CAR.KIA_EV6, fingerprint, [], False, False)
+      assert bool(CP.flags & HyundaiFlags.CANFD_HDA2) == hda2
 
     # radar available
     for radar in (True, False):
       fingerprint = gen_empty_fingerprint()
       if radar:
         fingerprint[1][RADAR_START_ADDR] = 8
-      CP = CarInterface.get_params(CAR.HYUNDAI_SONATA, fingerprint, car_fw, False, False)
+      CP = CarInterface.get_params(CAR.HYUNDAI_SONATA, fingerprint, [], False, False)
```

Comment:
Why this change?

Reply:
`car_fw` was only relevant for the `hda2` feature detection test. Since the `car_fw` is no longer relevant for determining HDA2, we should be able to just set an empty list here.